src/views/Profile.js

```diff
@@ -0,0 +1,340 @@
+                style={{
+                  backgroundImage: require("../assets/img/lifestyle.jpg")
+                }}
+              />
+              <div className="tex-content">
+                <h3>Only I can change my life. No one can do it me</h3>
+                <div className="tex-content__body">
+                  Bacon ipsum dolor sit amet bresaola shoulder ribeye jerky
+                  tongue andouille kevin meatloaf fatback shank bacon turkey
+                  turducken spare ribs chuck.
+                </div>
+                <div className="icons">
+                  <div className="left-icons">
+                    <img
+                      src={require("../assets/img/like-icon.svg")}
+                      alt="likes"
+                      className="likes"
+                    />
+                    <div className="numbers">3,844</div>
+                    <img
+                      src={require("../assets/img/dislike-icon.svg")}
+                      alt="dislikes"
+                      className="dislikes"
+                    />
+                    <div className="numbers">3,844</div>
+                    <img
+                      src={require("../assets/img/comment-icon.svg")}
+                      alt="comments"
+                      className="comments"
+                    />
+                    <div className="numbers">3,844</div>
+                  </div>
+                  <div className="right-icons">
+                    <img
+                      src={require("../assets/img/bookmark-icons.svg")}
+                      alt="bookmark"
+                      className="bookmark"
+                    />
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div className="article-card">
+              <div className="avatar-wrapper">
+                <img
+                  src={require("../assets/img/avatar-1.jpg")}
+                  alt="Avatar"
+                  className="avatar medium"
+                />
+                <div className="name_minutes medium">
+                  <span className="author_name">Joe Doe</span> <br />
+                  <span className="date_read_time">Nov 7,2018 . 3min read</span>
+                </div>
+              </div>
+              <div
+                className="main-article-img"
+                style={{
+                  backgroundImage: require("../assets/img/lifestyle.jpg")
+                }}
+              />
+              <div className="tex-content">
+                <h3>Only I can change my life. No one can do it me</h3>
+                <div className="tex-content__body">
+                  Bacon ipsum dolor sit amet bresaola shoulder ribeye jerky
+                  tongue andouille kevin meatloaf fatback shank bacon turkey
+                  turducken spare ribs chuck.
+                </div>
+                <div className="icons">
+                  <div className="left-icons">
```

Comment:
How about creating a single reusable component?

Reply:
@abayo-luc!
Thanks for the suggestions. However, I think this will be implemented later on during the project. When will be working on the feature that uses those components.